contentmgr/replication.go

```diff
@@ -111,32 +111,25 @@ type stagingZoneReadiness struct {
 	ReadinessReason string `json:"readinessReason"`
 }
 
-type contentStagingZone struct {
-	ZoneOpened time.Time `json:"zoneOpened"`
-
-	Contents []util.Content `json:"contents"`
-
-	MinSize int64 `json:"minSize"`
-	MaxSize int64 `json:"maxSize"`
-
-	CurSize int64 `json:"curSize"`
-
-	User uint `json:"user"`
-
-	ContID   uint   `json:"contentID"`
-	Location string `json:"location"`
-
-	IsConsolidating bool `json:"is_consolidating"`
-
-	Readiness stagingZoneReadiness `json:"readiness"`
-
-	lk sync.Mutex
+type ContentStagingZone struct {
+	ZoneOpened      time.Time            `json:"zoneOpened"`
+	Contents        []util.Content       `json:"contents"`
+	MinSize         int64                `json:"minSize"`
+	MaxSize         int64                `json:"maxSize"`
+	CurSize         int64                `json:"curSize"`
+	User            uint                 `json:"user"`
+	ContID          uint                 `json:"contentID"`
+	Location        string               `json:"location"`
+	IsConsolidating bool                 `json:"is_consolidating"`
```

Comment:
Is this something we return as part of an HTTP response? if not, lets use `isConsolidating` instead. If yes, leave it as is.

Reply:
It is not, I can fix it